xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -390,6 +390,18 @@
       "thumbnail": { "type": "string" }
     }
   },
+  "Library.Fields.Role": {
+    "extends": "Item.Fields.Base",
+    "items": { "type": "string", "enum": [ "title", "thumbnail" ] }
+  },
+  "Library.Details.Role": {
+    "extends": "Item.Details.Base",
+    "properties": {
+      "roleid": { "$ref": "Library.Id", "required": true },
+      "title": { "type": "string" },
+      "thumbnail": { "type": "string" }
```

Comment:
A silly. I had a syntax bug, did some re-copying to try and find it and the thumbnails sneeked back in. Will fix of course.